core/cas-server-core-web/src/main/java/org/apereo/cas/web/pac4j/CasSecurityInterceptor.java

```diff
@@ -0,0 +1,37 @@
+package org.apereo.cas.web.pac4j;
+
+import java.util.List;
+import org.pac4j.core.config.Config;
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.engine.DefaultSecurityLogic;
+import org.pac4j.core.exception.HttpAction;
+import org.pac4j.springframework.web.SecurityInterceptor;
+
+/**
+ * This is {@link CasSecurityInterceptor}.
+ *
+ * @author Arnold Bergner
+ * @since 5.2.0
+ */
+public class CasSecurityInterceptor extends SecurityInterceptor {
```

Comment:
Are we sure core-web is the best spot for this? Isn't this specifically a pac4j thing?

Reply:
True. I'd say a new core package then?

Reply:
How about `cas-server-support-pac4j-core`?

Reply:
Ah ok. I thought the pac4j support modules were specifically for delegation. I'm not super familiar with the project structure. I'll move it there then.